app/main.py

```diff
@@ -23,7 +23,12 @@
 def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
     result = None
     for cmd, method_type in commands:
+        if cmd not in commands_dict:
+            raise ValueError(f"Unknown command: {cmd}")
+        if method_type not in commands_dict[cmd]:
+            raise ValueError(f"Unknown method_type: {method_type}")
```

Comment:
This will immediately raise error on the first iteration because for example "form-data" method type is not present in display, but maybe it's in serializer command?